src/coreclr/vm/interoplibinterface_objc.cpp

```diff
@@ -15,6 +15,112 @@
 
 #define OBJC_MSGSEND "objc_msgSend"
 
+namespace
+{
+    BOOL g_ReferenceTrackerInitialized;
+    ObjCBridgeNative::BeginEndCallback g_BeginEndCallback;
+    ObjCBridgeNative::IsReferencedCallback g_IsReferencedCallback;
+    ObjCBridgeNative::EnteredFinalizationCallback g_TrackedObjectEnteredFinalizationCallback;
+
+    // Right now the begin/end values are defined to be
+    // positive for begin and negative for end. The actual
+    // value isn't important. Since CoreCLR is calling this
+    // during a gen2 that will be used but it isn't defined
+    // at this point. See documentation for further details.
+    const int BeginValue = 2;
+    const int EndValue = -BeginValue;
+
+    // Defined handle types for the specific object uses.
+    const HandleType InstanceHandleType{ HNDTYPE_REFCOUNTED };
```

Comment:
Nit: This alias is used in exactly one place. Use `HNDTYPE_REFCOUNTED` directly in that one place?